Samples/iOS-Swift/iOS-Swift/AppDelegate.swift

```diff
@@ -12,6 +12,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 return event
             }
             options.debug = true
+            options.tracesSampleRate = 1.0
+            options.tracesSampler = { _ in
+                return 1.0
+            }
```

Comment:
Docs say you should use only one of the two. I suggest:
```suggestion
            // Sampling 100% - In Production you probably want to adjust this
            options.tracesSampleRate = 1.0
```